capa/features/extractors/ghidra/helpers.py

```diff
@@ -263,9 +251,34 @@ def is_zxor(insn: ghidra.program.database.code.InstructionDB) -> bool:
     return all(n == operands[0] for n in operands)
 
 
+def handle_thunk(addr: ghidra.program.model.address.Address):
+    """Follow thunk chains down to a reasonable depth"""
+    ref = addr
+    for _ in range(THUNK_CHAIN_DEPTH_DELTA):
+        thunk_jmp = getInstructionAt(ref)  # type: ignore [name-defined] # noqa: F821
+        if thunk_jmp:
+            for i in range(thunk_jmp.getNumOperands()):
+                if OperandType.isAddress(thunk_jmp.getOperandType(i)):
+                    ref = thunk_jmp.getAddress(i)
```

Comment:
Using this loop we always choose the last operand that is an address. Can we skip the loop and jump straight to checking if the last operand is an address?